code/modules/power/singularity/singularity.dm

```diff
@@ -30,9 +30,10 @@
 
 	var/chained = 0//Adminbus chain-grab
 
-/obj/singularity/New(loc, var/starting_energy = 50, var/temp = 0)
+/obj/singularity/New(loc, var/starting_energy = 50, var/temp = 0, var/log = FALSE)
 	//CARN: admin-alert for chuckle-fuckery.
-	admin_investigate_setup()
+	if(log)

```

Comment:
Would argue against this. Bad OOP and allows for weird slip-ups.

Override `/obj/singularity/proc/admin_investigate_setup` for miniballs instead.